src/test/java/edu/tamu/cap/service/FedoraServiceTest.java

```diff
@@ -32,6 +33,7 @@
 import edu.tamu.cap.model.response.Version;
 import edu.tamu.cap.service.repositoryview.FedoraService;
 
+@ActiveProfiles("test")
 @ExtendWith(SpringExtension.class)
```

Comment:
`@SpringBootTest` already includes this, please remove `ExtendWith(SpringExtension.class)`.